tools/serverpod_cli/lib/src/commands/create.dart

```diff
@@ -33,7 +34,21 @@ class CreateCommand extends ServerpodCommand {
 
   @override
   Future<void> run() async {
-    var name = argResults!.arguments.last;
+    var rest = argResults?.rest;
+
+    if (rest == null || rest.isEmpty) {
+      log.info('Project name missing.');
```

Comment:
Should this not be `log.error`?

Reply:
Yes, I think you are right. Wasn't really sure how we should classify it. Will fix! 👍